Gopkg.toml

```diff
@@ -41,6 +41,10 @@ required = [
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1",
     ]
 
+[[constraint]]
+  name = "github.com/openshift/api"
+  revision = "8e476cb7322e59919cbb6482fd076ec5a214df25"
```

Comment:
This is the specific commit that has both Failing and Degraded.  Latest master already has failing removed, so this will be a multi-step process.